play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
+automatons you construct.
+At level 6, the Feat "With Friends Like These . . ." is automatically unlocked.
+Weapon Proficiencies: Pistols, Shotguns, Carbines
+Armor Proficiencies: Concealed, Recon, Light, Medium.
+Starting Wealth: $850 + 100*Luck + 1730
 Class Feats: 
------------------------
-A little Controlling - 
-You gain an additional permanant control point
-|
-|
-|
-V
-Control Phreak -
-You gain an additional permanant control point [max of 5 permanant points]
-----------------------
-Repairman -
-you gain a 10% chance to stabalize a crashing robotic ally and a +10% to 
-repair checks.
-|
-|
-|
-V
-Overbuilt -
-Any time you build a construction, you may spend an additional 15 nanites to 
-give that construction 50 additional health.
-
-"Priest" - you''ve lived your life in the deepest recesses of the holonet, and 
-have come out alive, and strong. You can figure out how a security system works
-, hack into systems, or even hack your allys'' nanites to perform better.
-Passives: Gain PC use [HACKING] at 10 OR Knowledge [NANITES] at 10
-Weapon Proficiencies: Pistols, SMGs.
-Armour Proficiencies: Concealed, Recon, Light, Medium.
-Starting Wealth: $1700+100*Luck - 650
-Class Feats
-----------------------
-Exploit weakness -
-You have a basic understanding of networks and computers, and can hack your 
-way into a system with ease, and have accrued many basic hacks onto your thumb
-drive belt. +20% against Firewalls.
-|
-|
-|
-V
-Your skill and library have grown substantially. +30% against Firewalls, +15% 
-against Info Nodes and Interaction Nodes.
-----------------------
-Nanite Hacking -
-You''ve had some cross-training in the medical field, or maybe you just tried 
-plugging a cord into your brain. You know how to communicate with Nanites 
-present in friends and enemies. DC 50 Knowledge [Nanites] grants a +20% to ally
-''s next roll, or -10% on an opponent. Touch Effect.
-|
-|
-|
-V
-Nanites have almost become second nature to you. DC 50 Knowledge [Nanites] 
-grants a +30% to ally''s next roll, or -15% on an opponent. Knowledge [Nanites]
-/ 2 meters range.
-
-"Pilot" - You have a yearning to test the limits of the vehicles you operate. 
+	-----------------------
+	A little Controlling - 
+	You gain an additional permanent control point
+	|
+	|
+	V
+	Control Phreak -
+	You gain an additional permanent control point [max of 5 permanent points]
+	----------------------
+	Repairman -
+	you gain a 10% chance to stabilize a crashing robotic ally and a +10% to 
+	repair checks.
+	|
+	|
+	V
+	Overbuilt -
+	Any time you build a construction, you may spend 15 Nanites to give that 
+	construction 50 additional health points.
+======================================
+
+
+=====   PRIEST   =====
+You've lived your life in the deepest recesses of the holonet, and have come out 
+alive and strong. You can figure out how a security system works, hack into systems,
+or even hack your allies' Nanites to perform better.
+Class Primary Attribute: Intelligence
```

Comment:
I'm not necessarily asking for a change here, just making a note. If Medic switches to Int as I proposed above, I recognize that that might create something of an Intelligence bloat in the bigger picture. So, I would note here that I could see moving priests to Perception as a primary attribute while keeping Int in their spell descriptions as a efficacy gauge, in the same style quality/quantity argument I used in the Medic's case. After all, you do have to see the weakness before you can hack at it.